### PR TITLE
[LF] Remove agreeement text from LF parser.

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
part of removing agreement text.
(See https://docs.google.com/document/d/14z_OOaeqXjfKrQubkk6Gt1bwHIYnzxzd1xQiarZICKk/edit#bookmark=id.tdh4ghxn547)

### CODE REVIEW

File: security-evidence.md [hunk 1]
```diff
@@ -32,163 +32,163 @@
...
-- Evaluation order of successful fetch_interface of a non-cached global contract: [EvaluationOrderTest.scala](daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala#L2651)
-- Evaluation order of successful lookup_by_key of a cached global contract: [EvaluationOrderTest.scala](daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala#L3004)
-- Evaluation order of successful lookup_by_key of a local contract: [EvaluationOrderTest.scala](daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala#L3081)
-- Evaluation order of successful lookup_by_key of a non-cached global contract: [EvaluationOrderTest.scala](daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala#L2923)
+- Evaluation order of create with authorization failure: [EvaluationOrderTest.scala](daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala#L573)
```
Comment (hubert-da) [inline security-evidence.md:35]: I'm not sure what this one is, but I trust you.

File: daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala [hunk 2]
```diff
@@ -1179,6 +1171,11 @@ class TransactionCoderSpec
     )
   }
 
+  private[this] def makePartyFresh(party: Party, create: Node.Create): IdString.Party = {
+    val contractParties = create.stakeholders ++ create.keyOpt.fold(Set.empty[Party])(_.maintainers)
+    Iterator.iterate(party)(p => Party.assertFromString(p + "_")).filterNot(contractParties).next()
```
Comment (hubert-da) [inline daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala:1176]: 👍🏻

**Review by hubert-da — APPROVED** [review]

File: security-evidence.md (hunk 1 above)
Comment (remyhaemmerle-da) [reply security-evidence.md:35]: it is an auto-generated file that list some tests

Comment (remyhaemmerle-da) [reply security-evidence.md:35]: `./fmt.sh` will modified it if need.

File: daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala [hunk 3]
```diff
@@ -826,8 +816,10 @@ class TransactionCoderSpec
           time,
           data.Bytes.fromByteString(salt),
         )
-        val nonMaintainerSignatories = instance.nonMaintainerSignatories + party
-        val nonSignatoryStakeholders = instance.nonSignatoryStakeholders + party
+        val party_ = makePartyFresh(party, create)
```
Comment (remyhaemmerle-da) [inline daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala:819]: This fixes a flak

**Pull request merged**